spec/system/qualifications/user_gives_feedback_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Feedback", type: :system do
+  include CommonSteps
+  include QualificationAuthenticationSteps
+
+  scenario "User gives feedback", test: %i[with_stubbed_auth with_fake_quals_api] do
+    given_the_qualifications_service_is_open
+    and_i_am_signed_in_via_identity
+    and_i_click_on_feedback
+    then_i_see_the_feedback_form
+    when_i_press_send_feedback
+    then_i_see_validation_errors
```

Comment:
There are several invocations of `then_i_see_validation_errors` following this one. You aren't resubmitting the form, so these steps aren't testing a state change.

Suggestion: test for the validation errors once, then simply complete and submit the form.